src/main/java/com/redhat/cloud/custompolicies/app/model/FullTrigger.java

```diff
@@ -47,10 +52,36 @@ public FullTrigger(Policy policy, boolean generatePseudoId) {
     trigger.name = policy.name;
     trigger.description = policy.description;
     trigger.enabled = policy.isEnabled;
-    conditions = new ArrayList<>(1);
     var cond = new Condition();
     cond.expression = policy.conditions;
     conditions.add(cond);
+    storeActions(trigger, policy);
+  }
+
+  private void storeActions(Trigger trigger, Policy policy) {
+    if (policy.actions == null) {
+      return;
+    }
+
+    String[] actionsIn = policy.actions.split(";");
+    for (String actionIn : actionsIn) {
+      var ta = new TriggerAction();
+      if (actionIn.trim().isEmpty()) {
+        continue;
+      }
+      if (actionIn.toLowerCase().startsWith("email")) {
+        ta.actionPlugin = "email";
+      } else if (actionIn.toLowerCase().startsWith("webhook")) {
+        ta.actionPlugin = "hooks"; // The legacy hooks apps
+        String endpoint = actionIn.substring(actionIn.indexOf(' ')+1);
+        ta.properties.put("endpoint_id",endpoint);
```

Comment:
Right, I did not notice this on our review session. But the old webhooks application does not work this way, we can't define the target destination. Instead, the target destination is all the activated webhooks in the old webhooks app.  So there's no parameters at all.

Reply:
Not even an id or something to identify which webhook we want to call?

Reply:
Oh that's a good point, it doesn't look like we can specify which webhook to trigger in the end  :man_facepalming: https://github.com/RedHatInsights/notifications-backend
CC: @jeromemarc

Reply:
@josejulio Nope, nothing. That's one of my gripes with it, it adds another "rules engine" on top of the existing and has just bad rules.

It's simply all or nothing for the old one.